omero-dev-runtests/buildomero.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Setup and run omero
+
+set -eux
+
+virtualenv /home/omero/omero-virtualenv --system-site-packages
+
+# Get pip to download and install requirements:
+
+set +o nounset
+source ~omero/omero-virtualenv/bin/activate
+set -o nounset
+
+pip install --upgrade pip
+pip install --upgrade 'Pillow<3.0'
+pip install --upgrade omego
+
+
+## download source
```

Comment:
If that is too complicated to maintain I can add ZIPURL the same like in https://github.com/ome/ome-docker/blob/master/omero-web/omeroweb.sh#L16